.gitignore

```diff
@@ -7,9 +7,9 @@ zappa_settings.json
 
 ### GOV.UK Frontend ###
 app/static/src/css*
-app/static/fonts*
```

Comment:
You will need to delete 
`app/static/fonts`
and
`app/static/images`
from your local repo and run `./build.sh` again once this is merged in